tests/test_annotation_cache.yml

```diff
@@ -13,15 +13,15 @@
       should_exist: false
 
 - name: Fail to locate VEP cache
-  command: nextflow run main.nf -profile test_cache,annotation --vep_cache s3://annotation-cache/vep_cache/ --vep_cache_version 1 --tools vep --input false --build_only_index --outdir results
+  command: nextflow run main.nf -profile test,annotation --vep_cache s3://annotation-cache/vep_cache/ --vep_cache_version 1 --tools vep --input false --build_only_index --outdir results
```

Comment:
So sometimes the argument following `--input` is interpreted as a path and at other times as a boolean?  I don't find that so nice, but I guess it doesn't matter.